### DISCUSSION TITLE
GONE: add / remove bookmark on the current page

### DISCUSSION THREAD

**Discussion by richo67** [koreader/koreader#8351] [Q&A] [answered]
Tapping on right top corner of the screen, use to create bookmark for the page, not anymore.

I am missing this for rather long time. It use to be there, but not anymore. I looked on documentation and [wiki ](https://github.com/koreader/koreader/wiki/KOReader-Gestures#addremove-bookmark-on-the-current-page) as well. I found it almost everywhere. It only doesn't work on my kindle paperwhite 4.

Did I set something to disable it?
Thankyou for your kind help.

**Comment by Frenzie** [comment, marked as answer]
In settings → taps & gestures → gesture manages → tap corner → top right corner

**Reply by richo67** [reply]
For those who have same challenge as I, in case you  do not have above mention menu, please do following as @Frenzie wrote lower:

> In that case:
> 
> Tools → More tools → Plugin management
> 
> And enable gestures. (And then restart.)

**Comment by richo67** [comment]
@Frenzie thanks for the quick answer, but I have no gesture manager in that menu. Running 2021.07 version now.

![IMG_20211021_155055](https://user-images.githubusercontent.com/9590148/138292375-d09c8663-0d49-44bb-b26a-4e48f91339e3.jpg)

**Reply by Frenzie** [reply]
In that case:

Tools → More tools → Plugin management 

And enable gestures. (And then restart.)

**Reply by richo67** [reply]
Thank you that was it!
I can not mark your last comment as answer ( no button "Mark as answer") there, strange.

**Reply by cwittern** [reply]
Hi, I have the same problem as the OP. However, even after enabling gestures in Plugin management and restarting, there is still no "gesture manager" menu, so no way to enable adding bookmarks.  This is KOreader 21.10 on Android (I got it to work in the same version on a Kobo Forma).

**Reply by Frenzie** [reply]
Are people that enthusiastic about disabling things? :) Does it show as enabled in plugin management?

**Reply by cwittern** [reply]
Yes, the plugin is shown as enabled, but the menu items are not available.

**Reply by Frenzie** [reply]
Did you also play around with custom menus?

**Reply by Frenzie** [reply]
PS Go to the main menu (hamburger, top right) → help → report a bug and attach the generated crash.log file.

**Reply by cwittern** [reply]
> Did you also play around with custom menus?

Sorry, but I have no ideas what these are.

**Reply by cwittern** [reply]
> PS Go to the main menu (hamburger, top right) → help → report a bug and attach the generated crash.log file.

I generated a crash.log file and can view it from within KOReader, but when I USB connect to the PC (Ubuntu 20.4), I can't see the file in that location.  Is there a way to transfer the file from within KOReader?

**Reply by Frenzie** [reply]
There isn't, but you should be able to open the same location in your device's file manager.
